calculator.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no, target-densitydpi=medium-dpi"/> 
+    <title>Blue Calculator</title>
+    <style>
+      table {
+       border: 2px solid black; 
+       margin: 10px;
+       width: 100px;
+       padding: 20px;
+       border-collapse: collapse;
+      }
+      
+      input {
+        text-align: right;
+        border: none;
+      }
+
+      input:focus {
+        outline: none;
+      }
+
+      th, td {
+       border: 1px solid black;
+       padding: 5px 10px;
+       text-align: center;
+      }
+
+      .a1{
+        background-color: gray;
+        color: white;
+      }
+
+      .a2{
+        background-color: whitesmoke;
+      }
+
+      .a3{
+        background-color: orange;
+        color: white;
+      }
```

Comment:
```suggestion
      .bg-grey {
        background-color: gray;
      }
      
      .font-white {
        color: white;
      }
      
      .bg-smoke{
        background-color: whitesmoke;
      }
      
      .bg-orange {
        background-color: orange;
      }
```

디자인 라이브러리처럼 재활용성이 좋네요!

이런 식으로 작성하면 다시 사용할 때 어떤 역할을 하는지 더 파악하기 쉬워질 것 같아요 😄